### DISCUSSION TITLE
Can't fetch any configuration from NVR nor Camera

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#339] [Questions]
Sorry about this topic, 

As I can't get register telegram account, and I didn't find there is any discussions or issues on github: https://github.com/OpenIPC/configurator.

So I choose here to ask. I download latest version, and do as 1/2/3. Finally, I found that there is no file downloaded from camera or nvr.
Is there any thing I have to configure?

PS: ssh to those ip with root/12345 is OK.

![图片](https://github.com/svpcom/wfb-ng/assets/1296171/147d4e54-f517-4c3d-8915-ead2212a2b95)
![图片](https://github.com/svpcom/wfb-ng/assets/1296171/0f922490-3eb9-4975-81d8-c9773847fb8d)
![图片](https://github.com/svpcom/wfb-ng/assets/1296171/cf497f00-6677-42cd-881a-3260f5051fc3)

**Comment by svpcom** [comment]
I'm not OpenIPC developer. Please open issue in OpenIPC github project

**Reply by svpcom** [reply]
https://github.com/OpenIPC/firmware
